frontend/src/components/common/SimpleTable.tsx

```diff
@@ -155,6 +263,13 @@ export default function SimpleTable(props: SimpleTableProps) {
                     {...otherProps}
                   >
                     {label}
+                    {sort &&
+                    <ColumnSortButtons
+                      isIncreasingOrder={isIncreasingOrder}
+                      isDefaultSorted={Boolean(defaultSortingColumn)}
```

Comment:
We are always passing `true` here (unless we have `defaultSortingColumn` as `0`).
Shouldn't we instead pass whether this column is the default one (so we should the buttons' according to that)?

I think what we want for each `ColumnSortButtons` is:
Is this column the currently sorting one?
 Yes -> then show the Up/Down according to the required state (by default is Increasing Order)
 No -> show both buttons so the user can choose

When changing to a different column sorting, the new order shouldn't have to do with the current one (because we're talking about different columns).
We can still manage the `isIncreasingOrder` state outside of the sort buttons as you're doing if you prefer, but we should reset the state when switching to a different column (i.e. would be decided in `sortClickHandler` above).

Reply:
So Boolean(defaultSortingColumn) always return true but returns false only if the column index passed is 0 or this prop wasn't passed in that case undefined i think this is what we want no?